File: onboarding/vaclav-onboarding-main/src/netglade_onboarding_flutter/lib/satellite_telemetry.dart
class SatelliteTelemetry {
  final int telemetryId;
  final int altitude;
  final int timestamp;
  final int temperature;
  final int velocity;
  final int radiation;

  const SatelliteTelemetry({
    required this.telemetryId,
    required this.altitude,
    required this.timestamp,
    required this.temperature,
    required this.velocity,
    required this.radiation,
  });
  factory SatelliteTelemetry.fromJson(Map<String, dynamic> json) {
    return SatelliteTelemetry(
        telemetryId: json['telemetryId'] as int,
        altitude: json['altitude'] as int,
        timestamp: json['timestamp'] as int,
        temperature: json['temperature'] as int,
        velocity: json['velocity']as int,
        radiation: json['radiation'] as int,);
  }
}

File: onboarding/vaclav-onboarding-main/src/Netglade.Onboarding.DummyDevice/Models/SatelliteTelemetry.cs
using System.Text.Json.Serialization;
using NetGlade.Onboarding.DummyDevice.Authentication;


namespace NetGlade.Onboarding.DummyDevice.Models;

public partial class SatelliteTelemetry
{
    public int TelemetryId { get; set; }

    public long Altitude { get; set; }

    public long Timestamp { get; set; }

    public int Temperature { get; set; }

    public int Velocity { get; set; }

    public int Radiation { get; set; }
    [JsonIgnore]
    public ICollection<ApplicationUser> ApplicationUsers { get; set; } = new List<ApplicationUser>();
}
